docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -101,19 +101,76 @@ Boolean variables
 =================
 
 Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False`` and so on. The matching of valid strings is case insensitive.
-While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
+Documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings. However, you can use any of the following values:
+
+Native boolean
+--------------
+
+Ansible treats these values as native booleans:
 
 .. table::
    :class: documentation-table
 
-   =============================================================================================== ====================================================================
-    Valid values                                                                                    Description
-   =============================================================================================== ====================================================================
-    ``True`` , ``'true'`` , ``'t'`` , ``'yes'`` , ``'y'`` , ``'on'`` , ``'1'`` , ``1`` , ``1.0``     Truthy values
+   ================= =================
+    Valid values      Description
```

Comment:
This is not really 'ansible native' but YAML that uses `true` and `false` and the rest of the list below, it would be 'more true' that `True` and `False` are the 'native' options as Ansible evaluation happens inside Python and those are the 'Python native booleans', this also depends on context as inside a `{{ }}` expression it is Jinja doing the evaluation, which is again using Python under the hood.